pandas/core/arrays/datetimelike.py

```diff
@@ -196,6 +196,71 @@ def astype(self, dtype, copy=True):
             return self._box_values(self.asi8)
         return super(DatetimeLikeArrayMixin, self).astype(dtype, copy)
 
+    # ------------------------------------------------------------------
+    # ExtensionArray Interface
+    # TODO:
+    #   * _from_sequence
+    #   * argsort / _values_for_argsort
+    #   * _reduce
+
+    def unique(self):
```

Comment:
I suppose this closes  https://github.com/pandas-dev/pandas/issues/23586 then? Can you confirm and add a "closes #23586" to the original issue?